src/mon/OSDMonitor.cc

```diff
@@ -6690,6 +6688,9 @@ int OSDMonitor::prepare_command_pool_set(const cmdmap_t& cmdmap,
 	return -EINVAL;
       }
       p.set_pg_num_pending(n, pending_inc.epoch);
+      // force luminous clients to resend their ops, since they
+      // don't understand pg_num_pending changes form a new interval
+      p.last_force_op_resend_luminous = pending_inc.epoch;
```

Comment:
should make sure we have some older client + mimic cluster with split/merge thrashing tests

Reply:
:+1: